i18n/translations/fr/form-builder.json

```diff
@@ -1158,5 +1158,24 @@
         "description": "Ajoutez une action pertinente spécifique pour aider les personnes à remplir le formulaire. Par exemple : « Supprimer l'adresse »"
       }
     }
+  },
+  "scheduleClosingPage": {
+    "linkText": "Programmer la fermeture du formulaire",
+    "dialog": {
+      "title": "Programmez le message de fermeture du formulaire",
+      "text1": "Fixez la date de fermeture",
+      "text2": "Décidez la date à laquelle vous souhaitez fermer le formulaire",
+      "datePicker": {
+        "month": "Mois",
+        "day": "Jour",
+        "year": "Année"
```

Comment:
In French, this should be in a different order:

[Day] [Month] [Year]

Reply:
Pete updated the UI.